requirements.txt

```diff
@@ -0,0 +1,42 @@
+ansible==6.7.0
```

Comment:
are these dependencies still needed even when we moved the new driver to the other repo?

Reply:
needed.